docs/editors-guide/externally-managed-content.md

```diff
@@ -0,0 +1,33 @@
+## Automatic Mondo Quality Control Checks for Externally Managed Content
+A subset of the extensive Mondo QC checks are automatically run on externally managed content. Any content from external providers that does not pass these QC checks is not included in Mondo until the content is updated and passes the QC checks. Reports of these QC checks will be available for review of any errors so that any needed changes can be made.
```

Comment:
Add "at the source" or "by the external providers"

"is not included in Mondo until the content is updated **at the source/by the data providers** and passes the QC checks."

Reply:
Maybe add something after the first sentence to explain that we do that so we don't block the update. 
Suggestion: 
**To limit delay due to QC failure,** any content from external providers (...)

Reply:
updated